wagtailmodelchoosers/client/components/BaseChooser.js

```diff
@@ -20,14 +20,9 @@ const defaultProps = {
 const propTypes = {
   initialValue: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
   updateInputValue: PropTypes.func.isRequired,
-  label: PropTypes.string.isRequired,
   initial_display_value: PropTypes.string.isRequired,
-  endpoint: PropTypes.string.isRequired,
-  value: PropTypes.any,
   required: PropTypes.bool.isRequired,
   display: PropTypes.oneOfType([PropTypes.string, PropTypes.array]).isRequired,
-  list_display: PropTypes.array.isRequired,
-  filters: PropTypes.array,
```

Comment:
Umm... `value` is indeed to be removed (I reckon it was renamed to `initialValue` and forgotten to be removed).

Regarding `label`, `endpoint`, `list_display` and `filters`, the `BaseChooser` doesn't indeed "need" them, but pass them down to the `ModelPicker` via `{...this.props}`. Because the `BaseChooser` is the interface the developer work with, it was a way to let him/her know everything which is needed for it to function correctly.

Any thoughts on that? Should it indeed be removed from here (at the expense of the developer having to know that `BaseChooser` uses a `ModelPicker` underneath with additional required `propTypes`) or left there (at the risk of getting out of sync)?